### DISCUSSION TITLE
Admonitions standard title label in translation file

### DISCUSSION THREAD

**Discussion by aborruso** [squidfunk/mkdocs-material#4599] [Ideas]
Hi,
if I add an admonition without title

```
!!! note 

    Lorem ipsum
```

I have the note icon and the English string "Note". But my site is in Italian, and it would be better to have in example "Nota".
I know I can add it as title in that Note, but I have dozens of notes without title and it would be convenient not to write for each one the generic title.

My proposal is to have a standard localized title for each type: note, abstract, info, tip, success, question, warning, failure, danger, bug, example and quote.

Thank you

**Comment by squidfunk** [comment]
Thanks for suggesting. Localization is template-specific, which is too late in the Markdown processing stage for Material for MkDocs. This would need to be implemented as part of Python Markdown, so you might ask their maintainers.
